src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import cn from 'classnames';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', {
+        'TodoInfo--completed': todo.completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+
+      {todo.user && <UserInfo user={todo.user} />}
```

Comment:
[CODE STYLE] - According to the checklist, you should prepare data in one place (App component) and pass it to child components instead of using `import` in non-root components. While you're not importing data directly in this component, it's implied that the user data should be attached to the todo object in the parent component. Ensure that the `todo` object with the user data is prepared in the App component or a similar higher-level component, and then passed down to this TodoInfo component.